cmd/tpm2_autounlock/tpm2_autounlock.sh

```diff
@@ -0,0 +1,176 @@
+#! /bin/bash
+
+# Default variables
+	# If password is stored in "defaultpassword" field, ensure the security of the file.
+defaultpassword="" 
+defaulttpmindex="0x1500016"
```

Comment:
According to:
https://trustedcomputinggroup.org/wp-content/uploads/RegistryOfReservedTPM2HandlesAndLocalities_v1p1_pub.pdf

The handle 0x1500016 is unassigned by TCG but reserved  for the Platform manufacturer. You should probably pick something in the unassigned by TCG for Owner range of:
Start: 0x0180000016
End: 0x01BFFFFF16

1800000
1BFFFFF

Reply:
Thanks.  Will stay away from that range.